backend/src/main/java/com/it/p/lodz/pl/masi/services/TestService.java

```diff
@@ -67,7 +68,16 @@ public EditResolveTestVersionDto getTestVersionById(long id) {
                 .map($ -> modelMapper.map($, EditResolveTestVersionDto.class))
                 .orElseThrow(TestVersionNotFoundException::new);
     }
-
+    public void assignTestToPosition(Long positionId, Long testId) {
+        var valuePosition = positionRepository.findById(positionId);
+        var valueTest = testRepository.findById(testId);
+        if (valuePosition.isPresent() && valueTest.isPresent()) {
+            TestEntity test = valueTest.get();
+            PositionEntity position = valuePosition.get();
+            position.addTestToPosition(test);
```

Comment:
W tym miejscu lepiej byłoby wykorzystać metodę `setPositionByPositionId` z `TestEntity` i później zapisać test, a nie pozycji. Będzie to też czytelniejsze, bo przypisujesz pozycję do testu, a nie test do pozycji.